frontend/www/js/omegaup/components/course/AssignmentList.vue

```diff
@@ -176,11 +194,19 @@ export default class CourseAssignmentList extends Vue {
       0,
       this.tests.splice(event.oldIndex, 1)[0],
     );
-    this.$emit(
-      'sort-tests',
-      this.courseAlias,
-      this.tests.map(test => test.alias),
-    );
+    this.testsOrderChanged = true;
+  }
+
+  saveNewOrder(type: string): void {
+    let param = null;
```

Comment:
```suggestion
    let param: string[] = [];
```

se puede hacer esto? así `param` no tiene que ser un `string[]|null`.